cli/flox/doc/flox-pull.md

```diff
@@ -7,53 +7,73 @@ header: "flox User Manuals"
 
 # NAME
 
-flox-pull -
+flox-pull - pull environment from FloxHub
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] pull [ `<options>` ] [ \--force ] [ ( -m | \--main) ]
+```
+flox [ <general-options> ] pull
+     [-d=<path>]
+     [-a]
+     [-r=<owner/name> | <owner/name> | [-f]]
+```
 
 # DESCRIPTION
 
-(`git`) Push or pull metadata to the environment's `floxmeta` repository,
-and in the `pull` case also proceed to render the environment.
-With this mechanism environments can be pushed and pulled between machines
-and within teams just as you would any project managed with `git`.
-
-With the `--force` argument flox will forceably overwrite either the
-upstream or local copy of the environment based on having invoked
-`push` or `pull`, respectively.
-
-With the `(-m|\--main)` argument `flox (push|pull)` will operate on the
-"floxmain" branch, pulling user metadata from the upstream repository.
-Cannot be used in conjunction with the `-e|\--environment` flag.
-
-With the `--no-render` argument `flox pull` will fetch and incorporate
-the latest metadata from upstream but will not actually render or create
-links to environments in the store. (Flox internal use only.)
+Pull an environment from FloxHub and create a local reference to it,
+or, if an environment has already been pulled, retrieve any updates.
+
+When pulling an environment for the first time, `-d` specifies the directory in
+which to create that environment.
```

Comment:
```suggestion
When pulling an environment for the first time, `-d` specifies the directory
in which to create that environment.
```


<details>
<summary>Backup for rule 6 i guess lol</summary>

![image](https://github.com/flox/flox/assets/7040031/8dc14877-3384-4900-8024-d89a8f8b5e58)

</details>

Reply:
As rule 6 is optional, I probably won't always follow it

Reply:
Yeah, almost all of them are optional, expect don’t break a hyphened word and do break after a sentence ^^ as I said in the other comment it’s  quite subjective still :/

Reply:
I would prefer we only review based on the required ones